queries/planet_osm_polygon.jinja2

```diff
@@ -147,7 +147,6 @@ FROM (
        mz_water_min_zoom < {{ zoom + 1 }}))
 {% elif zoom >= 7 %}
     ({{ bounds['polygon']|bbox_filter('way', 3857) }} AND
-      (mz_building_min_zoom < {{ zoom + 1 }} OR
```

Comment:
I think that deleting this line would leave the parentheses mismatched, and not sure why we wouldn't need the building min zoom. Perhaps this was a typo?

Reply:
Yes, it was a typo, good catch! Corrected in 0180c69.

I took buildings out here because this case would only get triggered for zoom level 7, and we know buildings won't be in the results at that zoom.